tests/test_align.py

```diff
@@ -45,6 +45,17 @@ def test_edit_distance():
         'total': 2
     }
 
+def test_edit_distance_sclite():
```

Comment:
Can you adjust the test so that it clearly shows there’s a different result depending on the flag setting?

Reply:
I am trying to come up with a simple example but can't think of anything. In this example itself, using uniform costs could give you `[(a,b),(b,c)]` or `[(a,*),(b,b),(*,c)]` both having cost=2, whereas sclite would always return the second alignment.